src/main/java/Task.java

```diff
@@ -0,0 +1,20 @@
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+    public Task completeTask() {
```

Comment:
While the naming is not wrong, this method can be better named as a setter i.e. `setComplete`, `setDone`, etc.